dts/arm/microchip/mec172x/mec172xnsz-pinctrl.dtsi

```diff
@@ -0,0 +1,1311 @@
+/*
+ * Copyright (c) 2021 Microchip Technology Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <dt-bindings/pinctrl/mchp-xec-pinctrl.h>
+
+&pinctrl {
+	/* ADC */
+	adc00_gpio200: adc00_gpio200 {
+		pinmux = < MCHP_XEC_PINMUX(0200, MCHP_AF1) >;
+		gpioph = < &gpio_200_236 >;
```

Comment:
This is both mixing GPIO and pinctrl, and, it duplicates information: 0200. Please just go with previous approach as it made more sense, IMHO.

Reply:
Previous approach involved using a gpio_ports[] table in the [pinctrl_mchp_xec.c](https://github.com/MicrochipTech/zephyr/blob/mec172x_pinctrl_mcreview1/drivers/pinctrl/pinctrl_mchp_xec.c)

Can you confirm we can revert back to that approach? 
This way the gpioph will be removed in the dtsi file as well; and there is no need for the gpio_xxx_xxx nodes within the pinctrl node as mentioned in the [comment](https://github.com/zephyrproject-rtos/zephyr/pull/41888#discussion_r808436043).

Please confirm.

Reply:
Removed phandle and reverted to original approach of using gpio_ports[].